src/openrct2/world/sprite.c

```diff
@@ -259,15 +259,13 @@ void sprite_clear_all_unused()
     spriteIndex = gSpriteListHead[SPRITE_LIST_NULL];
     while (spriteIndex != SPRITE_INDEX_NULL) {
         sprite = &get_sprite(spriteIndex)->unknown;
-        nextSpriteIndex = sprite->next;
-        previousSpriteIndex = sprite->previous;
-        memset(sprite, 0, sizeof(rct_sprite));
-        sprite->sprite_identifier = SPRITE_IDENTIFIER_NULL;
-        sprite->next = nextSpriteIndex;
-        sprite->previous = previousSpriteIndex;
+        sprite_reset(sprite);
         sprite->linked_list_type_offset = SPRITE_LIST_NULL * 2;
-        sprite->sprite_index = spriteIndex;
```

Comment:
It looks innocuous, but right before release I'd rather have this one line left in. There's no guarantee the sprite's index is set correctly. You can leave a comment explaining this oddity.